File: BetterIServ.Mobile/src/app/entities/substitution.ts
export interface Substitution {
  classes: string[];
  times: number[];
  type: string;
  representative: string;
  lesson: string;
  newLesson: string;
  room: string;
  teacher: string;
  description: string;
}

export interface UnitsData {
  date: Date;
  notifications: string[];
  substitutions: Substitution[];
}

File: BetterIServ.Backend/Entities/Substitution.cs
namespace BetterIServ.Backend.Entities; 

public struct Substitution {
    public string Class { get; set; }
    public int[] Times { get; set; }
    public string Type { get; set; }
    public string Representative { get; set; }
    public string Lesson { get; set; }
    public string Room { get; set; }
    public string Teacher { get; set; }
    public string Description { get; set; }
}
